get_started.sh

```diff
@@ -75,6 +75,12 @@ sudo apt-get install ros-kinetic-desktop-full
 sudo rosdep init
 rosdep update
 
+# Run rosinstall for other packages
+sudo mkdir /usr/share/ros
+sudo chmod a+rwx /usr/share/ros
+rosinstall .
+echo $DIR/setup.bash >> ~/.bashrc
```

Comment:
If there are new lines we need to add to the `.bashrc` file, they should be added to the `new_shell_config_lines` array on line 26. This prevents duplicate lines from being added to users config files (checks if they're already present), and lets us support multiple shells (bash, zsh, fish, etc.)

Reply:
I think we should probably be using the `setup.sh` script instead of `setup.bash`. In my experience this is the most universal option (ie. will usually work on bash and non-bash shells).

Reply:
Also, I would expect that we would want to run `source $DIR/setup.sh` instead of just `$DIR/setup.bash`? (The difference being that `source` will cause the script to be run in the *current* shell, rather then spawning a new shell to run it)

Reply:
One additional comment, since this has to do with installing dependencies for the repo, I think that everything but adding the `$DIR/setup.bash` file to the `.bashrc` file should probably be in the `install_depdencies` script.

Reply:
Comment 1: Will do
Comment 2: Sure
Comment 3: Forgot about that, will do
Comment 4: Ok